### DISCUSSION TITLE
Welcome to inspirers Discussions!

### DISCUSSION THREAD

**Discussion by Emmanuel-Melon** [Emmanuel-Melon/inspirers#92] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by samuel-ma** [comment]
Am gonna be the first to say something over here😁
Yes, am behind the scenes as usual fyi
Good luck👍 on finishing this project @Emmanuel-Melon 
The scale is just huge
Hope you still have the hunger and drive to proceed with getting this community out there
We need this....

**Reply by Emmanuel-Melon** [reply]
I appreciate you taking the time to pass by! We're actively working on this project. We have a UI designer now and we're basically waiting for her to finish some design components (cards, buttons, etc.). We're publishing a demo soon!
